examples/xml-attributes/xml_attributes.description

```diff
@@ -1,4 +1,7 @@
 // XML elements may have any number of attributes and any number
 // of namespace declarations that apply for that element. 
 // In Ballerina, both of these types are treated the same. 
-// Attributes are accessed from an XML sequence using the `@` postfix operator.
+// XML attribute access in ballerina is lax typed similar
+// to `json`, we can use field access expression (`.`) and
+// optional field access expressions (``?.`) to access teh
+// field of a xml value.
```

Comment:
The above needs to be slightly rephrased. Again, if we don't have an example for lax typing, users may be confused as to what lax typing is. @MaryamZi thoughts?

Reply:
There's a typo btw. `teh` -> `the`